error when using imagenette dataset

Hi, I'm getting the error below when I try to use the imagenette dataset.

```
AssertionError: Failed to construct dataset imagenette: Dataset imagenette cannot be loaded at version 0.1.0, only: 1.0.0.
```

Full traceback pasted below.

It seems to happen regardless of what built-in model I used: tried with a keras model and two torch models.

Attached config should replicate the error
[imagenette-resnet18.json.zip](https://github.com/twosixlabs/armory/files/8911708/imagenette-resnet18.json.zip)

Weights are downloaded from:
https://download.pytorch.org/models/resnet18-f37072fd.pth

e.g. with
```
curl -L -o ~/.armory/saved_models/resnet18-f37072fd.pth https://download.pytorch.org/models/resnet18-f37072fd.pth
```

Full traceback:  
```
2022-06-15 17:16:45  0s INFO     armory.eval.evaluator:__init__:32 Constructing Evaluator Object
2022-06-15 17:16:45  0s INFO     armory.eval.evaluator:__init__:63 Retrieving Environment Variables
2022-06-15 17:16:45  0s INFO     armory.docker.images:ensure_image_present:154 Retrieved canonical image name for twosixarmory/pytorch as twosixarmory/pytorch:0.15.3
2022-06-15 17:16:45  0s SUCCESS  armory.docker.images:is_image_local:125 found docker image twosixarmory/pytorch:0.15.3
2022-06-15 17:16:45  0s INFO     armory.docker.management:__init__:62 ARMORY Instance 4fd8c228ce created.
2022-06-15 17:16:45  0s INFO     armory.eval.evaluator:_run_config:272 Running evaluation script
2022-06-15 17:16:45  0s INFO     armory.delayed_imports:<module>:7 Importing and configuring torch, tensorflow, and art, if available. This may take some time.
2022-06-15 17:16:48.196901: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:936] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2022-06-15 17:16:48.197893: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:936] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2022-06-15 17:16:48.198516: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:936] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2022-06-15 17:16:48  2s INFO     armory.delayed_imports:<module>:30 Setting tf.config.experimental.set_memory_growth to True on all GPUs
2022-06-15 17:16:48  2s INFO     art.config:set_data_path:55 set ART_DATA_PATH to /tmp/.art/data
2022-06-15 17:16:48  2s INFO     art.config:set_data_path:55 set ART_DATA_PATH to /armory/saved_models/art
2022-06-15 17:16:49  3s WARNING  __main__:<module>:213 armory calling version 0.15.3 != armory imported version 0.0.0
2022-06-15 17:16:49  3s INFO     armory.data.utils:maybe_download_weights_from_s3:47 Using available resnet18-f37072fd.pth in Armory `saved_model_dir`
2022-06-15 17:16:52  7s ERROR    armory.scenarios.scenario:evaluate:397 Encountered error during scenario evaluation.
Traceback (most recent call last):
File "/opt/conda/lib/python3.8/runpy.py", line 194, in _run_module_as_main
return _run_code(code, main_globals, None,
│         │     └ {'__name__': '__main__', '__doc__': '\nMain script for running scenarios. Users will run a scenario by calling:\n    armory r...
│         └ <code object <module> at 0x7f7f4e2ed660, file "/opt/conda/lib/python3.8/site-packages/armory/scenarios/main.py", line 1>
└ <function _run_code at 0x7f809c4e7d30>
File "/opt/conda/lib/python3.8/runpy.py", line 87, in _run_code
exec(code, run_globals)
│     └ {'__name__': '__main__', '__doc__': '\nMain script for running scenarios. Users will run a scenario by calling:\n    armory r...
└ <code object <module> at 0x7f7f4e2ed660, file "/opt/conda/lib/python3.8/site-packages/armory/scenarios/main.py", line 1>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/main.py", line 228, in <module>
run_config(
└ <function run_config at 0x7f7f4d0cb040>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/main.py", line 144, in run_config
scenario.evaluate()
│        └ <function Scenario.evaluate at 0x7f7f4ce393a0>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7f809bbbb940>
> File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/scenario.py", line 388, in evaluate
self._evaluate()
│    └ <function Scenario._evaluate at 0x7f7f4ce39310>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7f809bbbb940>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/scenario.py", line 377, in _evaluate
self.load()
│    └ <function Scenario.load at 0x7f7f4ce52e50>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7f809bbbb940>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/scenario.py", line 271, in load
self.load_model()
│    └ <function Scenario.load_model at 0x7f7f4ce529d0>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7f809bbbb940>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/scenario.py", line 109, in load_model
model, _ = config_loading.load_model(model_config)
│              │          └ {'fit': False, 'fit_kwargs': {}, 'model_kwargs': {}, 'module': 'armory.baseline_models.pytorch.resnet18', 'name': 'get_art_mo...
│              └ <function load_model at 0x7f7f4d149a60>
└ <module 'armory.utils.config_loading' from '/opt/conda/lib/python3.8/site-packages/armory/utils/config_loading.py'>
File "/opt/conda/lib/python3.8/site-packages/armory/utils/config_loading.py", line 109, in load_model
model = model_fn(
└ <function get_art_model at 0x7f7f4ce39790>
File "/opt/conda/lib/python3.8/site-packages/armory/baseline_models/pytorch/resnet18.py", line 51, in get_art_model
wrapped_model = PyTorchClassifier(
└ <class 'art.estimators.classification.pytorch.PyTorchClassifier'>
TypeError: __init__() missing 1 required positional argument: 'nb_classes'
2022-06-15 17:16:53  8s ERROR    armory.docker.management:exec_cmd:105 command python -m armory.scenarios.main eyJfZGVzY3JpcHRpb24iOiAiSW1hZ2VuZXR0ZSB3aXRoIFJlc25ldDE4IHRyYWluZWQgb24gSW1hZ2VOZXQiLCAiYWRob2MiOiBudWxsLCAiYXR0YWNrIjogeyJrbm93bGVkZ2UiOiAid2hpdGUiLCAia3dhcmdzIjogeyJiYXRjaF9zaXplIjogMSwgIm5vcm0iOiAiaW5mIiwgImVwcyI6IDAuMDMxLCAiZXBzX3N0ZXAiOiAwLjAwNywgIm1heF9pdGVyIjogMjAsICJudW1fcmFuZG9tX2luaXQiOiAxLCAicmFuZG9tX2VwcyI6IGZhbHNlLCAidGFyZ2V0ZWQiOiBmYWxzZSwgInZlcmJvc2UiOiBmYWxzZX0sICJtb2R1bGUiOiAiYXJ0LmF0dGFja3MuZXZhc2lvbiIsICJuYW1lIjogIlByb2plY3RlZEdyYWRpZW50RGVzY2VudCIsICJ1c2VfbGFiZWwiOiB0cnVlfSwgImRhdGFzZXQiOiB7ImJhdGNoX3NpemUiOiA2NCwgImZyYW1ld29yayI6ICJudW1weSIsICJtb2R1bGUiOiAiYXJtb3J5LmRhdGEuZGF0YXNldHMiLCAibmFtZSI6ICJpbWFnZW5ldHRlIn0sICJkZWZlbnNlIjogbnVsbCwgIm1ldHJpYyI6IHsibWVhbnMiOiB0cnVlLCAicGVydHVyYmF0aW9uIjogImxpbmYiLCAicmVjb3JkX21ldHJpY19wZXJfc2FtcGxlIjogZmFsc2UsICJ0YXNrIjogWyJjYXRlZ29yaWNhbF9hY2N1cmFjeSJdfSwgIm1vZGVsIjogeyJmaXQiOiBmYWxzZSwgImZpdF9rd2FyZ3MiOiB7fSwgIm1vZGVsX2t3YXJncyI6IHt9LCAibW9kdWxlIjogImFybW9yeS5iYXNlbGluZV9tb2RlbHMucHl0b3JjaC5yZXNuZXQxOCIsICJuYW1lIjogImdldF9hcnRfbW9kZWwiLCAid2VpZ2h0c19maWxlIjogInJlc25ldDE4LWYzNzA3MmZkLnB0aCIsICJ3cmFwcGVyX2t3YXJncyI6IHt9fSwgInNjZW5hcmlvIjogeyJrd2FyZ3MiOiB7fSwgIm1vZHVsZSI6ICJhcm1vcnkuc2NlbmFyaW9zLmltYWdlX2NsYXNzaWZpY2F0aW9uIiwgIm5hbWUiOiAiSW1hZ2VDbGFzc2lmaWNhdGlvblRhc2sifSwgInN5c2NvbmZpZyI6IHsiZG9ja2VyX2ltYWdlIjogInR3b3NpeGFybW9yeS9weXRvcmNoIiwgIm51bV9ldmFsX2JhdGNoZXMiOiAzMCwgImV4dGVybmFsX2dpdGh1Yl9yZXBvIjogbnVsbCwgImdwdXMiOiAiYWxsIiwgIm91dHB1dF9kaXIiOiBudWxsLCAib3V0cHV0X2ZpbGVuYW1lIjogbnVsbCwgInVzZV9ncHUiOiB0cnVlLCAiZmlsZXBhdGgiOiAiZGF0YS9jb25maWdzL2ltYWdlbmV0dGVfcmVzbmV0NTAuanNvbiJ9LCAiZXZhbF9pZCI6ICIyMDIyLTA2LTE1VDE3MTY0NS4yNTA3MTYifQ== --num-eval-batches 30 --base64 did not finish cleanly
2022-06-15 17:16:53  8s INFO     armory.docker.management:stop_armory_instance:141 Stopping instance: 4fd8c228ce
2022-06-15 17:16:53  8s INFO     armory.eval.evaluator:_cleanup:119 deleting tmp_dir /home/ubuntu/.armory/tmp/2022-06-15T171645.250716
2022-06-15 17:16:53  8s INFO     armory.eval.evaluator:_cleanup:138 results output written to:
/home/ubuntu/repos/gard/eval5/results/2022-06-15T171645.250716/
ubuntu@ip-172-31-29-19:~/repos/gard/eval5$ nano data/configs/imagenette_resnet50.json
ubuntu@ip-172-31-29-19:~/repos/gard/eval5$ armory run data/configs/imagenette_resnet50.json
2022-06-15 17:19:48  0s INFO     armory.eval.evaluator:__init__:32 Constructing Evaluator Object
2022-06-15 17:19:48  0s INFO     armory.eval.evaluator:__init__:63 Retrieving Environment Variables
2022-06-15 17:19:48  0s INFO     armory.docker.images:ensure_image_present:154 Retrieved canonical image name for twosixarmory/pytorch as twosixarmory/pytorch:0.15.3
2022-06-15 17:19:48  0s SUCCESS  armory.docker.images:is_image_local:125 found docker image twosixarmory/pytorch:0.15.3
2022-06-15 17:19:49  0s INFO     armory.docker.management:__init__:62 ARMORY Instance 07c64829a4 created.
2022-06-15 17:19:49  0s INFO     armory.eval.evaluator:_run_config:272 Running evaluation script
2022-06-15 17:19:49  0s INFO     armory.delayed_imports:<module>:7 Importing and configuring torch, tensorflow, and art, if available. This may take some time.
2022-06-15 17:19:51.706353: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:936] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2022-06-15 17:19:51.707362: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:936] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2022-06-15 17:19:51.708000: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:936] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2022-06-15 17:19:51  2s INFO     armory.delayed_imports:<module>:30 Setting tf.config.experimental.set_memory_growth to True on all GPUs
2022-06-15 17:19:51  2s INFO     art.config:set_data_path:55 set ART_DATA_PATH to /tmp/.art/data
2022-06-15 17:19:52  2s INFO     art.config:set_data_path:55 set ART_DATA_PATH to /armory/saved_models/art
2022-06-15 17:19:52  3s WARNING  __main__:<module>:213 armory calling version 0.15.3 != armory imported version 0.0.0
2022-06-15 17:19:52  3s INFO     armory.data.utils:maybe_download_weights_from_s3:47 Using available resnet18-f37072fd.pth in Armory `saved_model_dir`
2022-06-15 17:19:56  7s INFO     art.estimators.classification.pytorch:get_layers:1158 Inferred 1 hidden layers on PyTorch classifier.
2022-06-15 17:19:56  7s INFO     armory.scenarios.scenario:load_dataset:205 Loading test dataset imagenette...
2022-06-15 17:19:56  7s ERROR    armory.scenarios.scenario:evaluate:397 Encountered error during scenario evaluation.
Traceback (most recent call last):
File "/opt/conda/lib/python3.8/runpy.py", line 194, in _run_module_as_main
return _run_code(code, main_globals, None,
│         │     └ {'__name__': '__main__', '__doc__': '\nMain script for running scenarios. Users will run a scenario by calling:\n    armory r...
│         └ <code object <module> at 0x7facd3e75660, file "/opt/conda/lib/python3.8/site-packages/armory/scenarios/main.py", line 1>
└ <function _run_code at 0x7fae1607bd30>
File "/opt/conda/lib/python3.8/runpy.py", line 87, in _run_code
exec(code, run_globals)
│     └ {'__name__': '__main__', '__doc__': '\nMain script for running scenarios. Users will run a scenario by calling:\n    armory r...
└ <code object <module> at 0x7facd3e75660, file "/opt/conda/lib/python3.8/site-packages/armory/scenarios/main.py", line 1>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/main.py", line 228, in <module>
run_config(
└ <function run_config at 0x7facd2c5f040>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/main.py", line 144, in run_config
scenario.evaluate()
│        └ <function Scenario.evaluate at 0x7facd29cd3a0>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7fae1574f940>
> File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/scenario.py", line 388, in evaluate
self._evaluate()
│    └ <function Scenario._evaluate at 0x7facd29cd310>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7fae1574f940>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/scenario.py", line 377, in _evaluate
self.load()
│    └ <function Scenario.load at 0x7facd29e6e50>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7fae1574f940>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/scenario.py", line 276, in load
self.load_dataset()
│    └ <function Scenario.load_dataset at 0x7facd29e6c10>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7fae1574f940>
File "/opt/conda/lib/python3.8/site-packages/armory/scenarios/scenario.py", line 206, in load_dataset
self.test_dataset = config_loading.load_dataset(
│                   │              └ <function load_dataset at 0x7facd2cdd9d0>
│                   └ <module 'armory.utils.config_loading' from '/opt/conda/lib/python3.8/site-packages/armory/utils/config_loading.py'>
└ <armory.scenarios.image_classification.ImageClassificationTask object at 0x7fae1574f940>
File "/opt/conda/lib/python3.8/site-packages/armory/utils/config_loading.py", line 73, in load_dataset
dataset = dataset_fn(batch_size=batch_size, framework=framework, *args, **kwargs)
│                     │                     │           │       └ {'epochs': 1, 'split': 'test', 'shuffle_files': False}
│                     │                     │           └ ()
│                     │                     └ 'numpy'
│                     └ 64
└ <function imagenette at 0x7facd2cd4f70>
File "/opt/conda/lib/python3.8/site-packages/armory/data/datasets.py", line 1071, in imagenette
return _generator_from_tfds(
└ <function _generator_from_tfds at 0x7facd2ccee50>
File "/opt/conda/lib/python3.8/site-packages/armory/data/datasets.py", line 412, in _generator_from_tfds
ds, ds_info = tfds.load(
│    └ <function load at 0x7facd376e280>
└ <module 'tensorflow_datasets' from '/opt/conda/lib/python3.8/site-packages/tensorflow_datasets/__init__.py'>
File "/opt/conda/lib/python3.8/site-packages/tensorflow_datasets/core/load.
py", line 322, in load
dbuilder = builder(name, data_dir=data_dir, try_gcs=try_gcs, **builder_kwargs)
│       │              │                 │          └ {}
│       │              │                 └ False
│       │              └ '/armory/datasets'
│       └ 'imagenette/full-size:0.1.0'
└ <function builder at 0x7facd376e160>
File "/opt/conda/lib/python3.8/site-packages/tensorflow_datasets/core/load.py", line 168, in builder
return cls(**builder_kwargs)  # pytype: disable=not-instantiable
│     └ {'config': 'full-size', 'version': '0.1.0', 'data_dir': '/armory/datasets'}
└ <class 'tensorflow_datasets.image_classification.imagenette.Imagenette'>
File "/opt/conda/lib/python3.8/site-packages/tensorflow_datasets/core/dataset_builder.py", line 941, in __init__
super().__init__(**kwargs)
└ {'config': 'full-size', 'version': '0.1.0', 'data_dir': '/armory/datasets'}
File "/opt/conda/lib/python3.8/site-packages/tensorflow_datasets/core/dataset_builder.py", line 196, in __init__
self._version = self._pick_version(version)
│               │    │             └ '0.1.0'
│               │    └ <function DatasetBuilder._pick_version at 0x7facd38155e0>
│               └ <tensorflow_datasets.image_classification.imagenette.Imagenette object at 0x7facb9eac0d0>
└ <tensorflow_datasets.image_classification.imagenette.Imagenette object at 0x7facb9eac0d0>
File "/opt/conda/lib/python3.8/site-packages/tensorflow_datasets/core/dataset_builder.py", line 276, in _pick_version
raise AssertionError(msg)
└ 'Dataset imagenette cannot be loaded at version 0.1.0, only: 1.0.0.'
AssertionError: Failed to construct dataset imagenette: Dataset imagenette cannot be loaded at version 0.1.0, only: 1.0.0.
2022-06-15 17:19:57  8s ERROR    armory.docker.management:exec_cmd:105 command python -m armory.scenarios.main eyJfZGVzY3JpcHRpb24iOiAiSW1hZ2VuZXR0ZSB3aXRoIFJlc25ldDE4IHRyYWluZWQgb24gSW1hZ2VOZXQiLCAiYWRob2MiOiBudWxsLCAiYXR0YWNrIjogeyJrbm93bGVkZ2UiOiAid2hpdGUiLCAia3dhcmdzIjogeyJiYXRjaF9zaXplIjogMSwgIm5vcm0iOiAiaW5mIiwgImVwcyI6IDAuMDMxLCAiZXBzX3N0ZXAiOiAwLjAwNywgIm1heF9pdGVyIjogMjAsICJudW1fcmFuZG9tX2luaXQiOiAxLCAicmFuZG9tX2VwcyI6IGZhbHNlLCAidGFyZ2V0ZWQiOiBmYWxzZSwgInZlcmJvc2UiOiBmYWxzZX0sICJtb2R1bGUiOiAiYXJ0LmF0dGFja3MuZXZhc2lvbiIsICJuYW1lIjogIlByb2plY3RlZEdyYWRpZW50RGVzY2VudCIsICJ1c2VfbGFiZWwiOiB0cnVlfSwgImRhdGFzZXQiOiB7ImJhdGNoX3NpemUiOiA2NCwgImZyYW1ld29yayI6ICJudW1weSIsICJtb2R1bGUiOiAiYXJtb3J5LmRhdGEuZGF0YXNldHMiLCAibmFtZSI6ICJpbWFnZW5ldHRlIn0sICJkZWZlbnNlIjogbnVsbCwgIm1ldHJpYyI6IHsibWVhbnMiOiB0cnVlLCAicGVydHVyYmF0aW9uIjogImxpbmYiLCAicmVjb3JkX21ldHJpY19wZXJfc2FtcGxlIjogZmFsc2UsICJ0YXNrIjogWyJjYXRlZ29yaWNhbF9hY2N1cmFjeSJdfSwgIm1vZGVsIjogeyJmaXQiOiBmYWxzZSwgImZpdF9rd2FyZ3MiOiB7fSwgIm1vZGVsX2t3YXJncyI6IHt9LCAibW9kdWxlIjogImFybW9yeS5iYXNlbGluZV9tb2RlbHMucHl0b3JjaC5yZXNuZXQxOCIsICJuYW1lIjogImdldF9hcnRfbW9kZWwiLCAid2VpZ2h0c19maWxlIjogInJlc25ldDE4LWYzNzA3MmZkLnB0aCIsICJ3cmFwcGVyX2t3YXJncyI6IHsibmJfY2xhc3NlcyI6IDEwfX0sICJzY2VuYXJpbyI6IHsia3dhcmdzIjoge30sICJtb2R1bGUiOiAiYXJtb3J5LnNjZW5hcmlvcy5pbWFnZV9jbGFzc2lmaWNhdGlvbiIsICJuYW1lIjogIkltYWdlQ2xhc3NpZmljYXRpb25UYXNrIn0sICJzeXNjb25maWciOiB7ImRvY2tlcl9pbWFnZSI6ICJ0d29zaXhhcm1vcnkvcHl0b3JjaCIsICJudW1fZXZhbF9iYXRjaGVzIjogMzAsICJleHRlcm5hbF9naXRodWJfcmVwbyI6IG51bGwsICJncHVzIjogImFsbCIsICJvdXRwdXRfZGlyIjogbnVsbCwgIm91dHB1dF9maWxlbmFtZSI6IG51bGwsICJ1c2VfZ3B1IjogdHJ1ZSwgImZpbGVwYXRoIjogImRhdGEvY29uZmlncy9pbWFnZW5ldHRlX3Jlc25ldDUwLmpzb24ifSwgImV2YWxfaWQiOiAiMjAyMi0wNi0xNVQxNzE5NDguNzk4NzUzIn0= --num-eval-batches 30 --base64 did not finish cleanly
2022-06-15 17:19:57  8s INFO     armory.docker.management:stop_armory_instance:141 Stopping instance: 07c64829a4
2022-06-15 17:19:57  8s INFO     armory.eval.evaluator:_cleanup:119 deleting tmp_dir /home/ubuntu/.armory/tmp/2022-06-15T171948.798753
2022-06-15 17:19:57  8s INFO     armory.eval.evaluator:_cleanup:138 results output written to:
/home/ubuntu/repos/gard/eval5/results/2022-06-15T171948.798753/
```

--- a/armory/data/datasets.py
+++ b/armory/data/datasets.py
@@ -1051,7 +1051,7 @@ def imagenette(
     preprocessing_fn = preprocessing_chain(preprocessing_fn, fit_preprocessing_fn)
 
     return _generator_from_tfds(
-        "imagenette/full-size:0.1.0",
+        "imagenette/full-size:1.0.0",
         split=split,
         batch_size=batch_size,
         epochs=epochs,

